org.eclipse.languageserver/src/org/eclipse/languageserver/LSPStreamConnectionProviderRegistry.java

```diff
@@ -67,15 +68,16 @@ private void initialize() {
 				String launchName = launchParts[1];
 				IContentType contentType = contentTypeManager.getContentType(contentTypeId);
 				if (contentType != null) {
-					ILaunchConfiguration config = LaunchConfigurationStreamProvider.findLaunchConfiguration(launchType, launchName);
+					ILaunchConfiguration config = LaunchConfigurationStreamProvider.findLaunchConfiguration(launchType,
```

Comment:
here too, I don't see how splitting the line makes it any better for maintainers.